controllers/suite_test.go

```diff
@@ -91,6 +96,12 @@ func TestAPIs(t *testing.T) {
 		panic(fmt.Sprintf("Environment var '%s' should be set to one of '%s' '%s' and '%s' values",
 			testDeploymentTypeEnvVarName, convergedDeploymentType, providerDeploymentType, consumerDeploymentType))
 	}
+	if testDeployment != convergedDeploymentType {
+		if os.Getenv(testRHOBSEndpointEnvVarName) == "" || os.Getenv(testRHssoTokenEndpointEnvVarName) == "" {
+			panic(fmt.Sprintf("Environment var '%s' and %s should be set",
+				testRHOBSEndpointEnvVarName, testRHssoTokenEndpointEnvVarName))
+		}
+	}
```

Comment:
- I don't see usage/need of this block, can this be removed?

Reply:
I am testing here the presence of the two env variables. If these were missing somehow the deployer will raise an error and fail to start similar to the block above it.

Reply:
> I am testing here the presence of the two env variables
- exactly, I feel this isn't required as you aren't using the env vars anywhere
- I see the values for these env vars are already defined in above `const` block and being used directly in `BeforeSuite` during creation of reconciler
- am I still missing anything here?

Reply:
@ezio-auditore I agree with @leelavg. In our repo, the env var are only consumed in `main.go` which then passes values to the controller. 
This is a test suite for the controller and as such, it makes no sense to test for env vars.

Reply:
Ack, removed.